docs/contributtion/development.md

```diff
@@ -0,0 +1,92 @@
+These include:
+
+- [Yarn workspaces][yarn-workspaces] for handling
+  dependencies across all packages
+- [Lerna][lerna] for publishing packages, tagging versions, and
+  more
+- [Make][make] for handling multi-technology tasks
+
+### System requirements
+
+In order for you to install all the dependencies in this project and successfully developing it, you'll need to:
+
+- [install Node.js][install-nodejs]
+- [install Yarn][install-yarn]
+- [install Docker][install-docker]
+- [install Make][install-make]
+
+### Starting up
+
+After you have set up all the requirement run the following command in your terminal:
+
+```bash
+yarn install
+# or
+make install
+```
+
+This will install all of the dependencies for every package in our project.
+In addition, it allows us to link between packages that we are developing.
+
+This strategy is particularly useful during development, and tooling like Lerna
+will pick up on when packages are linked in this way and will automatically
+update versions when publishing new versions of packages.
+
+Next up, you'll most likely want to build all of the package files so that
+things don't fail while you are working on a package.
+To do this, you can run the following command:
+
+```bash
+yarn build
+# or
+make build
+```
+
+Afterwards, you should be good to go!
+
+## Common tasks
+
+> Note: Most of the script are also available in the `Makefile` so you can use `make` instead of `yarn` if you prefer.
+> Run `make` or `make help` to see all available commands.
+> We are using Make to simplify and utilize the usage of all the scripts across the different technologies used in the project.
+
+While working on Spirit, here are some of the top-level tasks that you might want to run:
+
+| Command                          | Usage                                                                                                |
+| -------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| `yarn build`                     | Uses `lerna` to run the `build` script in each package                                               |
+| `yarn start`                     | Uses `lerna` to start development environments                                                       |
+| `yarn format`, `yarn format:fix` | Check if files have been formatted using Prettier, format files                                      |
+| `yarn test`                      | Uses `lerna` to run `test` script in each package (includes linting, unit testing and type checking) |
+
+In addition, you can use `yarn` to run `bin` files using the `yarn <bin>` syntax.
+For example, if you wanted to use `lerna` to run a script in every package you could do the following:
+
+```bash
+# Access $(yarn bin)/lerna and pass `run build` to the executable
+yarn lerna run build
+```
+
+[install-nodejs]: https://nodejs.org/en/download
+[install-docker]: https://docs.docker.com/get-docker/
```

Comment:
It looks like the links are almost alphabetically sorted :)

Reply:
Damn it! I was trying hard...